module/applications/actor/character-sheet-2.mjs

```diff
@@ -432,8 +432,7 @@ export default class ActorSheet5eCharacter2 extends ActorSheetV2Mixin(ActorSheet
         case "background": game.packs.get(CONFIG.DND5E.sourcePacks.BACKGROUNDS)?.render(true); break;
       }
     } else {
-      const result = await CompendiumBrowser.selectOne({ filters: { locked: { types: new Set([type]) } } });
-      if ( result ) this._onDropItemCreate(await fromUuid(result));
+      new CompendiumBrowser({ filters: { locked: { types: new Set([type]) } } }).render(true);
```

Comment:
For these specifically, I think we want to stick with drag-drop workflow as dragging from a compendium is how it worked before. The selection API remains in the compendium browser though for integration into summoning.